jmix-dynattr/dynattr-flowui/src/main/java/io/jmix/dynattrflowui/view/categoryattr/CategoryAttributesDetailView.java

```diff
@@ -278,7 +281,10 @@ protected void onInit(InitEvent event) {
     private void initDefaultEnumField() {
         defaultEnumValues.addAll(getEnumValues());
         defaultEnumField.setItems(defaultEnumValues);
-        defaultEnumField.addValueChangeListener(e -> getEditedEntity().setDefaultString(e.getValue()));
+        defaultEnumField.addValueChangeListener(e -> {
```

Comment:
why?

Reply:
don't bring outstaff code that not related with ticket

Reply:
also if lambda can be written into one line without brackets - don't use brackets